src/test/container-features/configs/lifecycle-hooks-advanced/.devcontainer/otter/devcontainer-feature.json

```diff
@@ -0,0 +1,22 @@
+{
+    "id": "otter",
+    "version": "1.2.3",
+    "options": {},
+    "updateContentCommand": "${featureRoot}/helper_script.sh updateContentCommand",
```

Comment:
Should we add the `featureRoot` property to the Feature interface?

Reply:
What do you mean by this? It is a part of the spec (https://github.com/devcontainers/spec/pull/196) and will make its way to the documentation, which interface?

Reply:
I think I misunderstood the use of this variable in first instance.

Clarifying question, are we expecting the Feature authors to use `${featureRootFolder}` in `devcontainer-feature.json` file? I think that's not the case from the [spec](https://github.com/devcontainers/spec/pull/196/files#diff-872faa37e7738b0b1885f9c60d4e5bad1462119460f6b4ebbdd95064143ec254R38-R48), but double checking

Reply:
If it's necessary then yes - https://github.com/devcontainers/spec/pull/196/files#diff-872faa37e7738b0b1885f9c60d4e5bad1462119460f6b4ebbdd95064143ec254R60

It's not required if the lifecycle command doesn't need to "reach into" the Feature (eg: the command is something on the PATH or relative to the workspace folder)